prefect_dbt/cli/configs/bigquery.py

```diff
@@ -86,8 +87,7 @@ class BigQueryTargetConfigs(TargetConfigs):
     project: Optional[str] = None
     credentials: GcpCredentials
 
-    @sync_compatible
-    async def get_configs(self) -> Dict[str, Any]:
+    def get_configs(self) -> Dict[str, Any]:
```

Comment:
Is this technically a breaking change? Would you see errors if you were previously awaiting this method?

Reply:
Yes this is a breaking change